src/papa_bringup/src/supervisor

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""Supervisor node.
+
+This node will receive all the cmd_vel messages sent to a given topic and 
+resend them thouugh another topic. A timeout is defined, if no cmd_vel messages
+are received after that time it will publish zero for all the speeds.
+
+This is implemented as a failsafe in case the robot lost comunication with the
+the control device. If that happens, it will stay in the same place, avoiding
+runaways.
+"""
+
+import numpy as np
+import rospy
+
+from geometry_msgs.msg import Twist
+
+
+class Supervisor:
+
+  def __init__(self):
+
+    rospy.init_node('supervisor', anonymous=True)
+
+    _input_topic = rospy.get_param('~input_topic', "cmd_vel")   
```

Comment:
Normally, the topic names are not taken from rosparam but they are remapped in the launchfile. (When doing gazebo plugins you use params in the SDF as it's the only option)